Gruntfile.js

```diff
@@ -1,9 +1,11 @@
 /**
  * @author Adam Meadows [@job13er](https://github.com/job13er)
- * @copyright 2015 Cyan, Inc. All rights reserved.
+ * @copyright 2015 Ciena Corporation. All rights reserved.
```

Comment:
This seems ironic since the Github organization is `cyaninc`. Wonder if/when this will change.